reconcile/dashdotdb_dvo.py

```diff
@@ -87,9 +87,11 @@ def _promget(self, url, query, token=None):
                   }
         if token:
             headers["Authorization"] = (f"Bearer {token}")
+        ssl_verify = False if is_private else True
```

Comment:
nit: Most likely you make `ssl_verify` an argument of `_promget` and get it via the `private` value so that you don't need to calculate it here. It would also feel more natural to pass that option to an http related function

Reply:
Excellent point. I moved this test out of the request method and made ssl_verify a "more natural" parameter.